tox.ini

```diff
@@ -1,9 +1,9 @@
 [tox]
-envlist = py{36,37}-dacite{110,120,150}-isort43-black20, py37-dacite150-isort{43,50,51}-black{19,20}, py{38,39}-dacite150-isort51-black20, development, latest
+envlist = py{36,37}-dacite{110,120,150}-isort43-black20, py37-dacite150-isort{43,50,51}-black{19,20}, py{38,39,310}-dacite150-isort51-black20, development, latest
 
 [testenv]
 deps =
-    pytest >=5.4.0,<6.0.0
+    pytest >=6.2.5,<7.0.0
```

Comment:
need to upgrade pytest for python 3.10

ref: https://stackoverflow.com/questions/69528842/an-error-while-trying-to-execute-tests-on-python-3-10-with-pytest